packages/common/src/editors/dateEditor.ts

```diff
@@ -186,7 +186,7 @@ export class DateEditor implements Editor {
           title: this.columnEditor && this.columnEditor.title || '',
           className: inputCssClasses.replace(/\./g, ' '),
           dataset: { input: '', defaultdate: this.defaultDate },
-          readOnly: this.columnEditor.readOnly === false ? false : true,
+          readOnly: this.columnEditor.editorOptions?.allowEdit === true ? true : false,
```

Comment:
probably easier to do `!!(this.columnEditor.editorOptions?.allowEdit)`

Reply:
passing on options to other options I'm typically careful since true is not the same as truthy but you're probably right here

Reply:
I've used this `!!(var)` for years, it's an easy shortcut to convert the output to a boolean

Reply:
thats right but `"false" == true`  and can happen relatively easy as consumer of the grid. chances are the user meant false just mistankely added quotes

Reply:
ahhh if that happens then yes I agree 😆

Reply:
@zewa666 ah this is probably why it fails, it looks like you forgot rename this readonly to `allowInput`